security/kubernetes-nodes.md

```diff
@@ -0,0 +1,280 @@
+---
+title: Protect Kubernetes nodes
+Description: Protect Kubernetes nodes with host endpoints managed by Calico
+---
+
+### Big picture
+
+Secure Kubernetes nodes with host endpoints managed by {{site.prodname}}.
+
+### Value
+
+{{site.prodname}} can automatically create host endpoints for your Kubernetes nodes. This means {{site.prodname}} can manage the lifecycle of host endpoints as your cluster evolves, ensuring nodes are always protected by policy.
+
+### Features
+
+This how-to guide uses the following Calico features:
+- **HostEndpoint**
+- **KubeControllersConfiguration**
+- **GlobalNetworkPolicy**
+
+### Concepts
+
+### Host endpoints
+
+Each host has one or more network interfaces that it uses to communicate externally. You can represent these interfaces in Calico using host endpoints and then use network policy to secure them.
+
+{{site.prodname}} host endpoints can have labels, and they work the same as labels on workload endpoints. The network policy rules can apply to both workload and host endpoints using label selectors.
+
+Automatic host endpoints secure _all_ of the hosts interfaces non-workload interfaces. They are created by setting `interfaceName: "*"`.
```

Comment:
Is "the hosts interfaces non-workload interfaces" right?  I thought they applied to traffic from pods to the host over e.g. the cali interfaces?  I think this perhaps should just say "all of the host's interfaces (i.e. in Linux, all the interfaces in the host network namespace)"?

Reply:
> I thought they applied to traffic from pods to the host over e.g. the cali interfaces?

Yes, traffic from workloads -> host are also enforced though the enforcement. I keep forgetting about apply-on-forward!

Reply:
2478717b4b77d62faf562c14ef2d2d5e0b713315